build.gradle

```diff
@@ -89,6 +86,6 @@ subprojects {
         project.android.dexOptions.preDexLibraries = rootProject.ext.preDexLibs
     }
 
-    project.plugins.withType(AppPlugin).whenPluginAdded(preDexClosure)
-    project.plugins.withType(LibraryPlugin).whenPluginAdded(preDexClosure)
+    project.plugins.withType(com.android.build.gradle.AppPlugin).whenPluginAdded(preDexClosure)
+    project.plugins.withType(com.android.build.gradle.LibraryPlugin).whenPluginAdded(preDexClosure)
```

Comment:
Why were these changes needed instead of the top imports?

Reply:
IDE seems to have trouble resolving those symbols in the top imports.

![image](https://user-images.githubusercontent.com/11519072/71461973-ddd6a280-2805-11ea-80c2-2317a1b87203.png)

Are you seeing the same thing?

It's not a big deal, just a bit annoying to see the red underline. Happy to revert if you strongly prefer the top imports 😃

Reply:
No strong opinions on this one, will leave it up to the repo owners :D